src/scripts/support/spotifyMetaData.js

```diff
@@ -28,6 +28,15 @@ var MetaData = {
     robot;
 var logger = require('./logger');
 
+/**
+ * Get ID from from Spotify URI by stripping out resource identifier e.g. 'spotify:track:'
+ * @param {string} uri
+ * @returns {string}
+ */
+function getId(uri) {
+    return uri.replace(/^spotify:[A-z]*:/, '');
```

Comment:
http://stackoverflow.com/questions/1658844/is-the-regular-expression-a-z-valid-and-if-yes-then-is-it-the-same-as-a-za-z#answer-1658859

I think we want either case-insensitive or [a-zA-Z]